bookshop/db/schema.cds

```diff
@@ -30,6 +31,16 @@ entity Genres : sap.common.CodeList {
   children : Composition of many Genres on children.parent = $self;
 }
 
+/** Hierarchically organized entity for Contents */
+entity Contents {
+    key ID       : UUID;
+        name     : String;
+        page     : Integer;
+        parent   : Association to Contents;
+        children : Composition of many Contents on children.parent = $self;
+        book     : Association to Books;
+}
+
```

Comment:
We should not pollute our central getting-started sample, but move that to the fiori / bookstore project.